packages/app-admin/src/plugins/Menu/Navigation/index.tsx

```diff
@@ -41,6 +39,22 @@ const Navigation = () => {
         return null;
     }, []);
 
+    const community = useMemo(() => {
```

Comment:
@BhuwanChandra after doing the suggested change. (please check the other comment)
You can simply use the following code snippet:
```js
const footerMenuPlugins =  getPlugins<AdminDrawerFooterMenu>("admin-drawer-footer-menu");
```
And then Iterate over these plugins and render the component. Like we are doing for menu plugins [here](https://github.com/webiny/webiny-js/blob/6de4de672a24de2e069c95419c4470fd4dbdf4a7/packages/app-admin/src/plugins/Menu/Navigation/index.tsx#L59)